core/test/integration/api/api_webhooks_spec.js

```diff
@@ -42,6 +43,7 @@ describe('Webhooks API', function () {
                 }).catch(checkForErrorType('ValidationError', done));
         });
 
+        // @TODO: this is a webhook model test for getByEventAndTarget (!)
```

Comment:
The webhook model test for this should check if `getByEventAndTarget` returns correct result or should directly try .add and see if duplication returns validation error(like we are using .add [here](https://github.com/TryGhost/Ghost/blob/master/core/test/unit/models/post_spec.js#L48))?